boa_engine/src/vm/graph.rs

```diff
@@ -0,0 +1,873 @@
+use std::mem::size_of;
```

Comment:
This module in general is missing some documentation at the module level and function/enum/struct level, to better understand what is it doing and how.